client/verta/verta/monitoring/summaries/queries.py

```diff
@@ -173,14 +177,17 @@ class SummarySampleQuery(object):
     --------
     .. code-block:: python
 
-        from datetime import datetime, timezone
-        from verta.monitoring.summary import SummaryQuery, SummarySampleQuery
+        from datetime import datetime, timedelta, timezone
+        from verta.monitoring.summaries.aggregation import Aggregation
+        from verta.monitoring.summaries.queries import SummaryQuery, SummarySampleQuery
 
         samples = Client().monitoring.summary_samples
+
         sample_query = SummarySampleQuery(
             summary_query=SummaryQuery(names=["Income Distributions"]),
             labels={"datasource": ["census2010", "census2020"]},
             created_after=datetime(year=2021, month=2, day=22, tzinfo=timezone.utc),
+            aggregation=Aggregation(timedelta(days=7), "sum")
```

Comment:
Light suggestion: I prefer maintaining trailing commas for simpler diffs if more lines are added in the future.
```suggestion
            aggregation=Aggregation(timedelta(days=7), "sum"),
```